packages/serverless/README.md

```diff
@@ -53,6 +53,19 @@ Sentry.AWSLambda.init({
 });
 ```
 
+#### Integrate Sentry using internal extension
+
+Another and much simpler way to integrate Sentry to your AWS Lambda function is to add an official layer.
+
+1. Choose Layers -> Add Layer.
+2. Specify an ARN: `arn:aws:lambda:us-west-1:TODO:layer:TODO:VERSION`.
+3. *(optional)* Go to Environment variables and add:
+   - `NODE_OPTIONS`: `-r @sentry/serverless/dist/auto`.
+   - `SENTRY_DSN`: `your dsn`.
+   - `SENTRY_TRACES_SAMPLE_RATE`: `1.0` if you want to enable tracing.
```

Comment:
I'd mention the docs here, as `1.0` is 100% sampling, which is not always desired